webapp/channels/src/components/admin_console/access_control/editors/table_editor/operator_selector_menu.tsx

```diff
@@ -13,8 +13,10 @@ import type {IDMappedObjects} from '@mattermost/types/utilities';
 
 import * as Menu from 'components/menu';
 
+import {OperatorLabel} from '../shared';
 import './selector_menus.scss';
 
+// TODO: Use Compass icons once a newer version is released
```

Comment:
> Do we have these icons defined in `@mattermost/compass-icons`? I don't know if that would contain math icons, but I would've assumed that UX would have already had these in there if they used them in designs

https://github.com/mattermost/mattermost/pull/30785#discussion_r2089093207 comment from @hmhealey 

I had a PR for that, it's merged. Waiting for the next release.